src/app/forms/form-layout/page.tsx

```diff
@@ -1,17 +1,40 @@
+'use client';
+import { useSelector, useDispatch } from 'react-redux';
+import { updateContactForm, updateSignInForm, updateSignUpForm } from '@/redux/forms/formSlice';
 import Breadcrumb from "@/components/Breadcrumbs/Breadcrumb";
-
 import { Metadata } from "next";
 import DefaultLayout from "@/components/Layouts/DefaultLayout";
 import SelectGroupOne from "@/components/SelectGroup/SelectGroupOne";
 import Link from "next/link";
+import { RootState } from '@/redux/store';
 
-export const metadata: Metadata = {
-  title: "Next.js Form Layout | TailAdmin - Next.js Dashboard Template",
-  description:
-    "This is Next.js Form Layout page for TailAdmin - Next.js Tailwind CSS Admin Dashboard Template",
-};
+// export const metadata: Metadata = {
+//   title: "Next.js Form Layout | TailAdmin - Next.js Dashboard Template",
+//   description:
+//     "This is Next.js Form Layout page for TailAdmin - Next.js Tailwind CSS Admin Dashboard Template",
+// };
 
 const FormLayout = () => {
+  const dispatch = useDispatch();
+  const contactForm = useSelector((state: RootState) => state.form.contactForm);
```

Comment:
Destructure contactForm as { firstName, lastName } and so on for all values. Then, remove use of contactForm from input fields like this:

` <input
                      type="text"
                      name="firstName"
                      value={firstName}
                      onChange={handleContactChange}
                      placeholder="Enter your first name"
                      className="w-full rounded border-[1.5px] border-stroke bg-transparent px-5 py-3 text-black outline-none transition focus:border-primary active:border-primary disabled:cursor-default disabled:bg-whiter dark:border-form-strokedark dark:bg-form-input dark:text-white dark:focus:border-primary"
                    />`